x-pack/plugins/observability_solution/metrics_data_access/common/inventory_models/types.ts

```diff
@@ -358,6 +358,8 @@ export const SnapshotMetricTypeKeys = {
   normalizedLoad1m: null,
   tx: null,
   rx: null,
+  txNew: null,
+  rxNew: null,
```

Comment:
Could we name  `rxNew` or `txNew` differently? I'm not sure what's better, but perhaps `tx_v2` or `rx_v2`, or `rx_rate_per_sec` or `tx_rate_per_sec`?

Reply:
Thanks, I used V2 as it will be easier to add new versions after in case we need to.